src/raw.rs

```diff
@@ -592,6 +592,52 @@ pub fn set_bitmap(data: &[u8]) -> SysResult<()> {
     Ok(())
 }
 
+
+///Set files to clipboard.
+pub fn set_file_list(paths: &str) -> SysResult<()> {
+    let final_str = paths;
+    let files_bytes: Vec<u16> = final_str.encode_utf16().collect();
+    let dropfiles_size = std::mem::size_of::<windows::Win32::UI::Shell::DROPFILES>() as u32;
+    // It seems files_bytes.len()*2 is valid, even for characters that span four bytes (as opposed
+    // to two bytes for, e.g., an ASCII character).
+    let path_size = files_bytes.len() * 2;
+    let dropfiles = windows::Win32::UI::Shell::DROPFILES {
```

Comment:
there is no point in creating this struct separately.
Just cast raw memory pointer to `*mut DROPFILES` and write into it using `ptr::write`